layer/ro_layer.go

```diff
@@ -43,6 +43,22 @@ func (rl *roLayer) TarStream() (io.ReadCloser, error) {
 	return rc, nil
 }
 
+func (rl *roLayer) TarStreamFrom(parent ChainID) (io.ReadCloser, error) {
```

Comment:
@dmcgowan This is from the refactor we talked about.
Seems fine for mountedLayer, but here it seems almost certainly not right based on the `TarStream()` implementation using tarsplit.

This "works", but I'm not certain what is "correct" here.